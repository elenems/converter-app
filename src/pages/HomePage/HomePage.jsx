import React from "react";
import Controls from "../../components/Controls";
import Graph from "../../components/Graph";
export default function HomePage() {
  return (
    <div id="homePage" className="row">
      <header>
        <Controls />
      </header>
      <main>
        <Graph />
      </main>
    </div>
  );
}
