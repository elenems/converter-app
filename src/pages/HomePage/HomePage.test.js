import HomePage from '../HomePage';
import {shallow} from 'enzyme';
import React from 'react';

describe("Home page", ()=>{
    it('renders', ()=>{
        const wrapper = shallow(<HomePage />).debug();
        expect(wrapper).toMatchSnapshot();
    })
})