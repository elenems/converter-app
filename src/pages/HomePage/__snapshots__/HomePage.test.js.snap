// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders 1`] = `
"<div id=\\"homePage\\" className=\\"row\\">
  <header>
    <Connect(Controls) />
  </header>
  <main>
    <Connect(Graph) />
  </main>
</div>"
`;
